tests/translate/translate_xtp_u.py

```diff
@@ -1,15 +1,17 @@
-import fv3core.stencils.xtp_u as xtpu
+import fv3core._config as spec
+import fv3core.stencils.xtp_u as xtp_u
 
 from .translate_ytp_v import TranslateYTP_V
 
 
 class TranslateXTP_U(TranslateYTP_V):
     def __init__(self, grid):
         super().__init__(grid)
+        self.in_vars["data_vars"]["u"] = {}
         self.in_vars["data_vars"]["c"]["serialname"] = "ub"
         self.in_vars["data_vars"]["flux"]["serialname"] = "vb"
 
-    def compute(self, inputs):
-        self.make_storage_data_input_vars(inputs)
```

Comment:
am i correct that this is wrapped by compute form storage now?

Reply:
yeah we just have another parent method we can use to reduce duplication